### PR TITLE
DOC: Wrong return type of np.random.choice and wrong variable name in parameter description.

### PR DESCRIPTION
**Pull request by MSeifert04**
This fixes some minor documentation issues in `np.random.choice`:

- The first parameter is called `a` and not `n` as indicated by the `np.arange(n)` comparison.
- The return value can be a scalar, a 1D array or an ND-array, currently the documentation suggests that it always returns a 1D array and the shape (possibly the size as well?).

### CODE REVIEW

File: numpy/random/mtrand/mtrand.pyx
Comment (eric-wieser) [inline numpy/random/mtrand/mtrand.pyx]: int isn't really correct here - it's either an arbitrary scalar (`size == None`), or an `ndarray`

File: numpy/random/mtrand/mtrand.pyx
Comment (eric-wieser) [inline numpy/random/mtrand/mtrand.pyx]: "As if `a` _were_ `np.arange(a)`". Backticks might be nice here too

Comment (MSeifert04) [inline numpy/random/mtrand/mtrand.pyx]: I omitted adding backticks because most of `random` documentation does not include them. And it's kind of nicer to read without code-formatting.

Comment (eric-wieser) [inline numpy/random/mtrand/mtrand.pyx]: I'd argue that at least `np.arange` should be backticked, but I don't care too much

**Review by eric-wieser — APPROVED** [review]

**Comment by MSeifert04** [comment]
Is there something that needs changing or is it just waiting for another review?

**Comment by eric-wieser** [comment]
Thanks Michael!

**Pull request merged**
